SeaPkg/Library/HashLibTpm2Raw/HashLibTpm2Raw.c

```diff
@@ -10,256 +10,12 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include <PiPei.h>
 #include <Library/BaseLib.h>
 #include <Library/BaseMemoryLib.h>
-#include <Library/Tpm2CommandLib.h>
+#include <Library/BaseCryptLib.h>

```

Comment:
Do we want to move along with this implementation? (vs. TPM hashing)

Reply:
Long term, I think TPM hashing makes sense. I think it scales better for something that is supposed to be generically reusable like the SEA binary.

I kept this in the PR because it's unclear exactly how much work is involved to do that and I wanted to get this branch functional minus all of the debug overhead.

Reply:
I might be the MbedTls code local to SeaPkg in commit that can be reverted (as opposed to it being in CryptoPkg) so we can stick with the main release/202311 Mu Basecore branch.

Reply:
okay, can do that

Reply:
Changes pushed.